src/main/java/com/github/onsdigital/babbage/configuration/Babbage.java

```diff
@@ -79,6 +82,14 @@ private Babbage() {
         exportSeverUrl = getValueOrDefault(HIGHCHARTS_EXPORT_SERVER_KEY, "http://localhost:9999/");
 
         mathjaxExportServer = getValue(MATHJAX_EXPORT_SERVER_KEY);
+
+        metricsEnabled = getStringAsBool(ENABLE_METRICS_KEY, "Y");
+
+        if (metricsEnabled) {
+            metricsPort = Integer.parseInt(getValueOrDefault(METRICS_PORT_KEY, "1234"));
```

Comment:
suggest set default port to a higher value

Reply:
done 👍